src/firebase/initialize.js

```diff
@@ -1,24 +1,20 @@
 import firebase from 'firebase';
 
 const config = {
-  apiKey: process.env.REACT_APP_API_KEY || 'AIzaSyB5hM6R8k3FzXuaIk_Hy3SGf0Owty2x6B4',
-  authDomain: process.env.REACT_APP_AUTH_DOMAIN || 'civic-app-cb9a8.firebaseapp.com',
-  databaseURL: process.env.REACT_APP_DATABASE_URL || 'https://civic-app-cb9a8.firebaseio.com',
-  projectId: process.env.REACT_APP_PROJECT_ID || 'civic-app-cb9a8',
-  storageBucket: process.env.REACT_APP_STORAGE_BUCKET || 'civic-app-cb9a8.appspot.com',
-  messagingSenderId: process.env.REACT_APP_MESSAGEING_SENDER_ID || '664717193548',
+  apiKey: process.env.REACT_APP_API_KEY || 'AIzaSyCZKRQR4Q1x5mFWD1kD8vTo0uiXYWgWqNo',
```

Comment:
So, uh...., which app is the right one?

Reply:
I think yours is the one we are going to switch to, since it is not under my personal account.  Just load the mock data from the mock data file into it and everything that is already in place should still work